reporting.md

```diff
@@ -0,0 +1,14 @@
+# Reporting bugs
+
+This topic covers reporting bugs in the AWS Command Line Interface (AWS CLI). If you’d like to make a feature request, follow the **Intake** section in the [Contribution process](contribution_process.md#intake). 
+
+To report a bug, perform the following steps:
+
+1. Perform initial troubleshooting to confirm this is not a support issue. If this is an error, do not open a GitHub issue, instead [request support](support.md).
+2. Before reporting a bug, check to see if there's an existing open [issue](https://github.com/aws/aws-cli/issues)or [pull request](https://github.com/aws/aws-cli/pulls) for the bug.
+3. If there isn't an existing issue, file a new GitHub issue. When creating a new issue, use one of the suggested issue  templates as issues may require different information. In general, the ideal bug report includes:
```

Comment:
It might be worth adding a quick link to opening a GitHub issue: https://github.com/aws/aws-cli/issues/new/choose. We have a template for bug report if that they should choose when filing the issue.